ouroboros-consensus/src/Ouroboros/Consensus/Protocol.hs

```diff
@@ -79,10 +78,10 @@ data Protocol blk where
 
   -- | Run PBFT against the real ledger
   ProtocolRealPBFT
-    :: Cardano.Genesis.Config
+    :: Genesis.Config
```

Comment:
Maybe we should switch to record syntax?

Reply:
I was thinking of it, but dared not to suggest, assuming important reasons for not doing so : -)

Reply:
It would make the `runProtocol` pattern match below a lot clearer.

Reply:
Are we not afraid of partial field accessors?

We had a discussion about them in a similar situation: https://github.com/input-output-hk/cardano-node/pull/147#discussion_r317308688

Reply:
In this case the fields make the code more understandable. IMO, one shouldn't use field accessor functions anyway (https://github.com/ghc-proposals/ghc-proposals/pull/160) 🙂 (Maybe with newtypes, alright).